plugins/utils/harpoon.nix

```diff
@@ -198,10 +200,21 @@ in {
       // cfg.extraOptions;
   in
     mkIf cfg.enable {
+      warnings = mkIf (!config.plugins.telescope.enable && cfg.enableTelescope) [
+        "Nixvim: The harpoon telescope integration needs telescope to function as intended"
+      ];
```

Comment:
Please use an assertion instead of a warning, see my previous comment about those

Reply:
apologies i forgot to make that change!